File: ProgramasJava/src/br/com/evertonsantos/secao06/Exercicio05.java
package br.com.evertonsantos.secao06;

import java.util.Scanner;

public class Exercicio05 {

	public static void main(String[] args) {
		float peso, exc, multa;
		Scanner teclado = new Scanner(System.in);
		
		System.out.println("Informe o peso pescado: ");
		peso = teclado.nextFloat();
		
		if(peso > 50) {
			exc = peso - 50;
			multa = exc * 4;
			System.out.printf("O peso excedente pescado é de %.2fKg, devendo pagar uma multa no total de R$%.2f", exc, multa);
		}else {
			exc = 0;
			multa = 0;
			System.out.println("Você não excedeu do peso limite; Multas= R$" + multa);
		}
		
		teclado.close();
	}
	}

File: ProgramasPython/secao06/exercicio05.py
peso= float(input("Informe o peso total de peixes pescados: "))

if peso > 50:
    e = peso - 50
    m = e * 4
    print("Peso total: {0:.2f}Kg".format(peso))
    print("Peso excendente: {0:.2f}Kg".format(e))
    print("Multa a ser paga: R${0:.2f}".format(m))
    
else:
    print("Você não excedeu o limite de peso, nao tendo que pagar multas.")    